packages/serverless/src/awslambda.ts

```diff
@@ -314,7 +316,12 @@ export function wrapHandler<TEvent, TResult>(
       clearTimeout(timeoutWarningTimer);
       transaction.finish();
       hub.popScope();
-      await flush(options.flushTimeout);
```

Comment:
as an argument, nullifying error on flushing for google functions
https://github.com/getsentry/sentry-javascript/blob/1bf988322a6adb931905305747f4fafb742002b1/packages/serverless/src/gcpfunction/events.ts#L57-L63

Reply:
for traditional expessjs/Koa
https://github.com/getsentry/sentry-javascript/blob/1bf988322a6adb931905305747f4fafb742002b1/packages/node/src/handlers.ts#L403-L409

Reply:
additionally, as i understand, 'flush' operation may be time consuming, so here it can affect lambda execution time even on successful invocation

Reply:
Afaik this will not work in the case of AWS, as it will freeze the process immediately after reaching return value of the function.
Why it works for GCP, is that there is an explicit `callback` that tells the runtime that it's done executing.

Reply:
yep, you are right, with freezing the process, we can't do anything after the event, only via Lambda Extension, which is not current case. btw, in this PR i only propose to ignore sentry errors

Reply:
@kamilogorek may i ask you to clarify what exact updates you requested

Reply:
I think I had a brain-fart for a second, as I was almost certain you changed from `await flush` to `void flush` 😅
It's all good! Thanks :)